blockchain/README.md

```diff
@@ -26,35 +26,51 @@ As described before: The Trubudget Blockchain is created one by a master, which
+| API_PROTO                   | no       | http          | The Protoll which should be used to connect to the master-node's api. (http, https)                                                                                                                                                                                                                                                                |
+| EXTERNAL_IP                 | no       |               | The IP address with which the current node can be reached. The variable is forwarded to the mutlichain dameon as `externalip` argument. <br>Example: If you have a VM running on 52.52.52.52 and you want to start a slave node from this VM to connect to a master running on 53.53.53.53, you set the `EXTERNAL_IP` to 52.52.52.52 on this node. |
+| LOG_LEVEL                   | no       | INFO          | Sets the lowest level to the pino logger that is printed to `STDOUT` with varying levels ranging from `trace` to `fatal`.                                                                                                                                                                                                                          |
+| MULTICHAIN_DIR              | no       | /root         | The path to the multichain folder where the blockchain data is persisted. For installations via `docker-compose`, this refers to the path within the docker container of the blockchain. For bare metal installations, this refers to the path on the machine the blockchain is running on.                                                        |
+| ORGANIZATION                | yes      | -             | In the blockchain network, each node is represented by its organization name. This environment variable sets this organization name. It is used to create the organization stream on the blockchain.                                                                                                                                               |
+| P2P_HOST                    | no       |               | The IP address of the blockchain node you want to connect to. When given, the node joins the existing network rather than creating its own chain.                                                                                                                                                                                                  |
+| P2P_PORT                    | no       | 7447          | The port on which the node you want to connect to has exposed the blockchain.                                                                                                                                                                                                                                                                      |
```

Comment:
_The port on which the node you want to connect to has exposed the blockchain._ to
_The port on which the node you want to connect to has to be exposed to the blockchain._

Reply:
This should be right. The blockchain has to be exposed via P2P_PORT. Your sentence describe the port has to be exposed.